components/infobox/commons/infobox_person.lua

```diff
@@ -206,6 +208,12 @@ function Person:_setLpdbData(args, links, status, personType, earnings)
 		links = links,
 		extradata = {},
 	}
+
+	_earnings.total = nil
```

Comment:
Don't really like this. Maybe return from Earning Module two values? One the total and a separate for the tableByYear?